cx_Freeze/samples/build_constants/hello.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+import sys
+from sys import stdout
+
+stdout.write('Hello from cx_Freeze\n')
+stdout.write('The current date is %s\n\n' %
+             datetime.today().strftime('%B %d, %Y %H:%M:%S'))
+
+stdout.write('Executable: %r\n' % sys.executable)
+stdout.write('Prefix: %r\n' % sys.prefix)
+stdout.write('Default encoding: %r\n' % sys.getdefaultencoding())
+stdout.write('File system encoding: %r\n\n' % sys.getfilesystemencoding())
+
```

Comment:
Why use 'stdout.write' here and print in another place? I think you can use print everywhere.

Reply:
It just came from the code in samples/simple that I copied.  Removed most of it and replaced the remaining ones with prints.